src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_VALUE = 100;
```

Comment:
How do you think what max number will be returned by random.nextInt(MAX_VALUE), where MAX_VALUE = 100?